npc/other/Global_Functions.txt

```diff
@@ -159,11 +159,8 @@ function	script	F_Rand	{
 //== Function F_Sex ========================================
 // Returns 1st argument if female, 2nd argument otherwise
 // Example: mes callfunc("F_Sex","What a beautiful lady!","What a handsome man!");
-function	script	F_SexMes	{
-	if (Sex == SEX_MALE)
-		return getarg(0);
-	else
-		return getarg(1);
+function	script	F_Sex	{
+	return getarg(Sex == SEX_MALE);
```

Comment:
```callfunc("F_Sex","What a beautiful lady!","What a handsome man!");```
`getarg(0)` - for female , result should be `What a beautiful lady`
`getarg(1)` - for male , result should be `What a handsome man!`

`SEX_FEMALE = 0`
`SEX_MALE = 1`

when `(Sex == SEX_MALE)`,
if your char's Sex = Female , which is value `0` .... the condition become `if ( 0 == 1 )`, which is `false`, and become `getarg(0)`
if your char's Sex = Male , which is value `1` .... the condition become `if ( 1 == 1 )`, which is `true`, and become `getarg(1)`

it create the correct output. I dont see how it cause issues to any of current scripts.